src/main/java/seedu/address/model/person/Person.java

```diff
@@ -42,6 +43,7 @@ public Person(Name name, Phone phone, Email email, Address address, Flag flag, S
         this.tags.addAll(tags);
         this.prevDateMet = prevDateMet;
         this.info = info;
+        this.scheduledMeeting = null;
```

Comment:
I don't think it is a good idea to pass around null as it can cause unexpected errors in the future. Maybe can try ``this.scheduledMeeting = new scheduledMeeting();`` 
Then this construction can create an empty meeting object that can be passed instead of null.